typings/index.d.ts

```diff
@@ -913,7 +915,7 @@ declare module 'discord.js' {
     public setName(name: string, reason?: string): Promise<GuildEmoji>;
   }
 
-  export class GuildMember extends PartialTextBasedChannel(Base) {
+  export class GuildMember extends (PartialTextBasedChannel(Base) & { lastMessageChannelID: Snowflake | null }) {
```

Comment:
Wouldn't adding `lastMessageChannelID` as regular field be better for readability?

Reply:
oh, i meant to fix that but forgot, not sure why i did it like that in the first place